exports/csv.php

```diff
@@ -73,6 +73,9 @@ function get_post_activity( $post_type ) {
     );
 }
 $data_type = isset( $_GET['type'] ) ? esc_url_raw( wp_unslash( $_GET['type'] ) ) : '';
+$data_type = str_replace( 'https://', '', $data_type );
+$data_type = str_replace( 'http://', '', $data_type );  // http://contacts
```

Comment:
@cairocoder01 in case you're wondering about this, I needed to do this bc the `$data_type` was being set to `http://contacts` or `http://group_activities` on Ln#75, and so it was falling into the `default` case of the `switch` statement each time

Reply:
That's because I mistakenly used the wrong escaping/sanitize function on the GET parameter on the line above. Instead of `esc_url_raw` (which expects the parameter to be a URL), change that to `sanitize_key` (see [Data Sanitization/Escaping docs](https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/))

Reply:
@cairocoder01 `sanitize_key` will return `httpcontacts` in my dev env, so we would need to edit the switch statement (which I don't think that we would want to do)

Reply:
Thx for sharing! I am slowly coming up to speed with WordPress best practices